packages/site-admin/.storybook/mock-registry/stores.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { store as coreDataStore } from '@wordpress/core-data';
+
+const stores = {
+	'WooCommerce Analytics': {
+		[ coreDataStore.name ]: {
+			name: 'WooCommerce Analytics',
+			description:
+				'WooCommerce Analytics is a powerful tool that helps you understand how your store is performing and how you can improve your store’s performance.',
+			'root/__unstableBase': { site_icon_url: './woo-logo.png' },
```

Comment:
Can we reuse existing logos from the repo instead of adding new copies?

Reply:
We can. I just wanted to isolate the story resources in its folder. I can update you if you think it would be better. 👍

Reply:
[Site Admin]: Improve SiteIcon component story #99710